src/System.Memory/src/System/SpanSortHelpers.KeysAndOrValues.cs

```diff
@@ -0,0 +1,779 @@
+﻿//// Licensed to the .NET Foundation under one or more agreements.
+//// The .NET Foundation licenses this file to you under the MIT license.
+//// See the LICENSE file in the project root for more information.
+
+//using System.Collections.Generic;
+//using System.Diagnostics;
+//using System.Runtime.CompilerServices;
+
+//#if !netstandard
+//using Internal.Runtime.CompilerServices;
+//#endif
+
+//using static System.SpanSortHelpersCommon;
+//using S = System.SpanSortHelpersKeysAndOrValues;
+
+//namespace System
+//{
+//    // TODO: This is my futile attempt to consolidate all variants into a 
```

Comment:
In that case, let's get rid of the file (feel free to file an issue as an enhancement for the backlog to revisit it if you think it is worthwhile).

Reply:
deleting it, yes